### DISCUSSION TITLE
How to make the report page show the full url name, path in the sections "Not Found URLs (404s)" and others graphs.

### DISCUSSION THREAD

**Discussion by karnizas85** [allinurl/goaccess#2503] [Q&A]
Hello,

Sorry if I am duplicating this question, I tried to look for it but did not find it.
How to make the report page show the full url name, path in the sections "Not Found URLs (404s)" and others graphs.
I think I need to change the logging format in /etc/goaccess.conf " log-format %h %^ [%d:%t %^] %^ "%v" "%m %U %H" %s %b "% R" "%u" %T %^ %^ %^ %^" to other type to display the result I want. Thank you for help..

**Comment by allinurl** [comment]
Please take a look at the [man page, examples](https://goaccess.io/man#examples) > virtual host. Essentially you need to append the virtual host to the request in order to see which virtual host the top urls belong to. #117 will address this. Stay tuned as it's coming soon!
